script/reverse_urls.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+
+from argparse import ArgumentParser
+from urllib.parse import urlsplit
+import re
+
+def reverse_url(url):
+    components = urlsplit(url)
+    hostname = components.netloc.partition(':')[0]
+    result = '.'.join(hostname.split('.')[::-1])
+    if components.path:
+        result += components.path
+    if components.query:
+        result += '.' + components.query
+    if components.fragment:
+        result += components.fragment
+
+    return re.sub(r'\W', '.', result).lower()
+
+
+def main():
+    parser = ArgumentParser(description="Take a text file of URLs (one per line) \
```

Comment:
Why not unify this script with [this other](https://github.com/pisa-engine/pisa/blob/master/script/generate_sorted_docids_mapping.py)? Maybe could be added an option like `--reverse-url`.

Reply:
These are scripts that perform different tasks, I prefer to keep them in separate files